pkgs/development/python-modules/mal/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub
+, buildPythonPackage, isPy34, isPy35
+, appdirs, requests, decorating
+}:
+
+# tests currently fail on Python 3.6, try adding isPy36 when the next version
+# after 0.3.2 is released
+assert (isPy34 || isPy35);
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.